src/geometry/manifold/manifold-applyops-minkowski.cc

```diff
@@ -107,27 +102,29 @@ std::shared_ptr<const Geometry> applyMinkowskiManifold(const Geometry::Geometrie
           t.stop();
           PRINTDB("Minkowski: decomposition took %f s", t.time());
         }
-        return part_points;
+        return std::move(part_points);
```

Comment:
iirc `return std::move` is not a good pattern as the compiler should automatically do that already.